### DISCUSSION TITLE
How do you start recording?

### DISCUSSION THREAD

**Discussion by ndudde67** [fuatakgun/eufy_security#356] [Q&A]
I've follow all the steps AFAIK, but I can't figure out what service/entity I'm supposed to use to start a camera's recording functionality. I've got my Eufy system to talk to HA and I can see when they are triggered and everything. I don't really care (for the moment) about seeing video footage in the HA app, but I would like to be able to have HA decide when to start and stop the cameras based on automations, not based on the Eufy system's arm/disarm state.

**Comment by fuatakgun** [comment]
If your camera is already streaming, check camera services to record it. I didn't implement anything specifically for recording, HA would take care of it.

**Reply by Zwer2k** [reply]
So far, I haven't figured out if and how it's possible to start recording so that it's saved on the docking station as usual. I'd like to use an external presence detector for this, as the internal motion detection starts and stops recording far too often, which means the camera is constantly clicking away in the evenings. Unfortunately, I haven't found any recent threads on this topic.
